docs/pages/x/api/date-pickers/index.md

```diff
@@ -4,27 +4,91 @@
 
 ## Components
 
-- [DateCalendar](/x/api/date-pickers/date-calendar/)
+### Pickers
```

Comment:
I added the unstable components and ordered everything because it was becoming impossible to read